CHANGES.rst

```diff
@@ -27,6 +27,9 @@ We still recommend checking out the new features and giving feedback in the repo
 Minor changes
 ~~~~~~~~~~~~~
 
+- Move sections in Table of Content of Reference guide.
+- Improve :py:class:`icalendar.prop.vDatetime` documentation. See `Issue #946 <https://github.com/collective/icalendar/issues/946>`_.
```

Comment:
I don't know why these two lines are included in your pull request, since they don't have anything to do with it. Please make sure your fork is up to date with `main`.

```suggestion
```